echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/scheduler/actions/pipeline/impl/MissedPipelineTriggerCompensationJob.groovy

```diff
@@ -197,11 +198,16 @@ class MissedPipelineTriggerCompensationJob implements ApplicationListener<Contex
         return
       }
 
-      def expr = new CronExpression(trigger.cronExpression)
-      expr.timeZone = dateContext.timeZone
+      try {
+        CronExpression expr = new CronExpression(trigger.cronExpression)
+        expr.timeZone = dateContext.timeZone
 
-      if (missedExecution(expr, lastExecution, dateContext.triggerWindowFloor, dateContext.now, pipeline)) {
-        pipelineInitiator.call(pipeline)
+        if (missedExecution(expr, lastExecution, dateContext.triggerWindowFloor, dateContext.now, pipeline)) {
+          pipelineInitiator.call(pipeline)
+        }
+      } catch (ParseException e) {
+        log.error("Error parsing cron expression (${trigger.cronExpression}) for pipeline ${pipeline.id}")
+        onOrcaError(e)
```

Comment:
Somewhat annoying that `onOrcaError` is going to dump a stack trace with a rather nondescript `log.error("Error retrieving latest pipeline executions", error)` message.

The metric increment makes sense but maybe we could do without that log output?